### PR TITLE
fix(client) fixed icon height in containers

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

fix for issue #41083 this is a fix for the containers not having a fixed height and icon sizes are standardized

### CODE REVIEW

**Comment by raisedadead** [comment]
@ahmadabdolsaheb do you mind giving us an approval for merge. Thanks.

**Comment by Sembauke** [comment]
> @ahmadabdolsaheb do you mind giving us an approval for merge. Thanks.

I think the thumbs-up is the approval.

**Comment by raisedadead** [comment]
> I think the thumbs-up is the approval.

Nope, it's not. It just means he has acknowledged my request. 

We have a certain internal policy about the QA. We request your patience while we get around to it, cheers.

**Comment by Sembauke** [comment]
> > I think the thumbs-up is the approval.
> 
> Nope, it's not. It just means he has acknowledged my request.
> 
> We have a certain internal policy about the QA. We request your patience while we get around to it, cheers.

I'm sorry I didn't know that I will wait on Ahmads approval.

**Comment by ojeytonwilliams** [comment]
Hey @Sembauke, there's nothing to apologise for - it's great that you're helping out with the UI.

The approval in question here is just an approving review.  So, assuming Ahmad's happy, he'll do that and we can merge this in.

File: client/src/components/landing/landing.css
Comment (ahmadabdolsaheb) [inline client/src/components/landing/landing.css:188]: We should not set the height or max-height for the buttons. Since the text will wrap for smaller screens, setting the height would squeeze the text and reduce padding.

I suggest setting the min-height to the height of the largest button, that way they would all look the same on desktop. In smaller screens the text will wrap and the buttons will have different larger sizes which is ok.

**Review by ahmaxed — CHANGES_REQUESTED** [review]
Thank you for your patience. Other than the following nitpick, the PR looks good.

File: client/src/components/landing/landing.css
Comment (Sembauke) [reply client/src/components/landing/landing.css]: All buttons were less than 80px due to reformatting the height of the icons. So I set the min-height to 80px.

**Comment by Sembauke** [comment]
Can I get a review from @ahmadabdolsaheb

**Review by ahmaxed — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks @Sembauke, the /learn page looks nicer now.

**Comment by Sembauke** [comment]
> Thanks @Sembauke, the /learn page looks nicer now.

Thanks

**Pull request merged**
